apps/client/src/intros/DakkapelIntro.js

```diff
@@ -1,38 +1,12 @@
 import { Heading } from "@datapunt/asc-ui";
 import React from "react";
 
-import { List, ListItem, OrderedList } from "../atoms";
-import ListInsideOrderedList from "../components/ListInsideOrderedList";
+import { List, ListItem } from "../atoms";
+import LocationIntro from "./shared/LocationIntro";
 
 export default () => (
   <>
-    <Heading forwardedAs="h3">Hoe het werkt:</Heading>
-    <OrderedList>
-      <ListItem>
-        U voert op de volgende pagina eerst het adres van het gebouw in.
-      </ListItem>
-      <ListItem>
-        Vervolgens krijgt u te zien:
-        <ListInsideOrderedList variant="bullet">
-          <ListItem>of het gebouw een monument is.</ListItem>
-          <ListItem>
-            of het gebouw in een beschermd stads- of dorpsgezicht ligt.
-          </ListItem>
-          <ListItem>welk bestemmingsplan er geldt.</ListItem>
-        </ListInsideOrderedList>
-      </ListItem>
-      <ListItem>
-        Wij stellen u een aantal vragen over het gebouw en de dakkapel.
```

Comment:
Now you're missing this text:
_Wij stellen u een aantal vragen over het gebouw **en de dakkapel**._

Reply:
That's correct.